### PR TITLE
Add synthetics type to manifest spec

### PR DESCRIPTION
**Pull request by justinkambic**
## What does this PR do?

I added a new `type` to the manifest spec, as we are introducing `synthetics`.

## Why is it important?

We are working on publishing a package for Elastic Synthetics and we need to provide this type for our data streams.

## Checklist

- [ ] ~I have added test packages to [`code/go/internal/validator/test/packages`](https://github.com/elastic/package-spec/tree/master/code/go/internal/validator/test/packages) that prove my change is effective.~
- [x] I have added an entry in [`versions/N/changelog.yml`](https://github.com/elastic/package-spec/blob/master/versions/1/changelog.yml).

## Related issues

-

### CODE REVIEW

File: versions/1/changelog.yml [hunk 1]
```diff
@@ -60,4 +60,8 @@
     link: https://github.com/elastic/package-spec/pull/116
   - description: Support Terraform deployment files. Enable "deploy" definitions for data stream
     type: enhancement
-    link: https://github.com/elastic/package-spec/pull/111
\ No newline at end of file
+    link: https://github.com/elastic/package-spec/pull/111
+  - description: Add "synthetics" to package type enum in manifest spec
```
Comment (ycombinator) [inline versions/1/changelog.yml:64]: Technically it's a data stream type, not a package type.

**Comment by ycombinator** [comment]
Also, after you've fixed the CHANGELOG, run `make update`. That should fix the error CI is complaining about right now.

**Comment by justinkambic** [comment]
> lso, after you've fixed the CHANGELOG, run make update. That should fix the error CI is complaining about right now.

Done in https://github.com/elastic/package-spec/pull/119/commits/80700ea417b553849fd8f857f832a1a0c166c170.

**Review by ycombinator — APPROVED** [review]
LGTM.

**Pull request merged**